portfolio/src/main/webapp/index.html

```diff
@@ -427,5 +427,19 @@ <h1>Hello Google Maps</h1>
   <!-- Iconify Icons -->
   <script type="text/javascript" src="js/iconify.min.js"></script>
 </body>
-
-</html>
\ No newline at end of file
+<div>
+  <head>
+    <title>Hello Google Maps</title>
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyC1A1a0VUuJkisJi-Aat9n21h8K7OxPbKQ"></script>
```

Comment:
I don't think the api key should be visible as plain text on github. The [maps api documentation](https://developers.google.com/maps/documentation/javascript/get-api-key) doesn't advise against having it on the client side though, maybe @abhatia2107 can help answer the question